internal/cli/config.go

```diff
@@ -47,11 +48,12 @@ func defaultSections() sections {
 		outputs:      false,
 		providers:    false,
 		requirements: false,
+		resources:    false,
 	}
 }
 
 func (s *sections) validate() error {
-	items := []string{"header", "inputs", "outputs", "providers", "requirements"}
+	items := []string{"header", "inputs", "outputs", "providers", "requirements", "resources"}
 	for _, item := range s.Show {
 		switch item {
 		case items[0], items[1], items[2], items[3], items[4]:
```

Comment:
Newly added case is missing here.

```suggestion
		case items[0], items[1], items[2], items[3], items[4], items[5]::
```